src/components/editor/PanelMain.tsx

```diff
@@ -218,10 +242,12 @@ export const PanelMain: React.FC = () => {
               </Button>
             </Tooltip>
           )}
-        </Flex>
+          <Text fontSize='xs' color="gray.400">{lastChangedText}</Text>
+          <Text fontSize='xs' color="red">{saveStateMessage}</Text>
```

Comment:
Same here, this editor doesn't have a concept of saved state, and many of places it's used do not either.